__tests__/app.test.js

```diff
@@ -109,56 +109,152 @@ describe("GET: /api/reviews/:review_id errors", () => {
           expect(comment).toHaveProperty("votes", expect.any(Number));
           expect(comment).toHaveProperty("created_at", expect.any(String));
-        })
         });
-    });
-    test("The returned comments array should be ordered with the most recent comments first!", () => {
-      return request(app)
+      });
+  });
+  test("The returned comments array should be ordered with the most recent comments first!", () => {
+    return request(app)
       .get("/api/reviews/3/comments")
       .expect(200)
       .then(({ body }) => {
-        const {comments} = body;
+        const { comments } = body;
         expect(comments).toBeSortedBy("created_at", { descending: true });
       });
-    })
-    test("When a Valid review ID is used but there are no comments for that ID a blank comments array is returned", () => {
-      return request(app)
+  });
+  test("When a Valid review ID is used but there are no comments for that ID a blank comments array is returned", () => {
+    return request(app)
       .get("/api/reviews/5/comments")
       .expect(200)
       .then(({ body }) => {
-        const {comments} = body;
-        expect(comments).toEqual([])
+        const { comments } = body;
+        expect(comments).toEqual([]);
       });
-    })
   });
+});
 
-  describe("GET: /api/reviews/:review_id/comments errors", () => {
-    test("Responds with a 404 custom error message when a valid but non existent review ID is input", () => {
-      return request(app)
-        .get("/api/reviews/15/comments")
-        .expect(404)
-        .then((res) => {
-          expect(res.text).toBe("Review_id not found");
-        });
-    });
-  })
-
-
-
+describe("GET: /api/reviews/:review_id/comments errors", () => {
+  test("Responds with a 404 custom error message when a valid but non existent review ID is input", () => {
+    return request(app)
+      .get("/api/reviews/15/comments")
+      .expect(404)
+      .then((res) => {
+        expect(res.text).toBe("Review_id not found");
+      });
+  });
+});
 
+describe("POST: 201 /api/reviews/:review_id/comments", () => {
+  test("Adds a comment with a request body object of username and body for a review to the comments data and returns the comment", () => {
+    const reqBody = {
+      username: "dav3rid",
+      body: "I hate werewolves, I would rather play something about vampires!",
+    };
+    return request(app)
+      .post("/api/reviews/3/comments")
+      .send(reqBody)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment[0].comment_id).toBe(7);
+        expect(comment[0].body).toBe(
+          "I hate werewolves, I would rather play something about vampires!"
+        );
+        expect(comment[0].review_id).toBe(3);
+        expect(comment[0].author).toBe("dav3rid");
+        expect(comment[0].votes).toBe(0);
+        expect(comment[0]).toHaveProperty("created_at", expect.any(String));
+      });
+  });
```

Comment:
Ideally the comment would be an object, rather than a single object in an array but the test looks good